packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -113,3 +113,7 @@ HostPlatform getCurrentHostPlatform() {
 
   return HostPlatform.linux_x64;
 }
+
+String getBuildDirectory() {
+  return config.getValue('build-dir') ?? 'build';
```

Comment:
👍  Agreed, outputs feels like a totally user-specific preference, not project-level configuration. Leaving as is.